### ISSUE TITLE
Divi Page Settings in Page/Post-editor

### ISSUE THREAD

**Issue by JLDokter** [WordPress/gutenberg#10937]
Without Gutenberg activated I see this:

<img width="310" alt="zonder gutenberg classic editor" src="https://user-images.githubusercontent.com/40452630/47351361-189bf280-d6b8-11e8-88f1-10c20dfd8acc.png">

With (only) Gutenberg activated I see this:

<img width="283" alt="met gutenberg - zonder classic editor" src="https://user-images.githubusercontent.com/40452630/47351470-50a33580-d6b8-11e8-8c8e-c56c74287cdd.png">

With both Gutenberg AND Classic editor activated, I see this:

<img width="300" alt="met gutenberg - met classic editor" src="https://user-images.githubusercontent.com/40452630/47351524-6dd80400-d6b8-11e8-9bda-2b19f710b06d.png">

So, with Gutenberg activated both the "Dot.navigation" and the "Show Title" options are not available.

**Comment by swissspidy** [comment]
Have you reached out to the theme developers about this?

A quick Google search says Divi has some initial Gutenberg support, so it's likely that they _removed_ these fields when Gutenberg is active.

**Comment by JLDokter** [comment]
Yes, I have submitted this issue to Elegant Themes first. But I don't know one which side the problem lies. I presume the theme is passive and that Gutenberg overrides the display of the fields. Therefor I decided to post this issue here also.

I don't think that Divi is the problem here, because there is no way to hide/make visible the Page/Post-title anywhere else. If Divi has decided to make this fields invisible, then they would have build the same possibility somewhere else. Now it is just disappeared.

NB: the possibility to hide/make visible the page/post title is essential when you want to create a custom template.
